test-harness/specs/security-scheme-validation-AC-27.oas2.txt

```diff
@@ -0,0 +1,46 @@
+====test====
+A defined 422 response is returned when security validation is passed.
+====spec====
+swagger: "2.0"
+paths:
+  /users:
+    get:
+      description: returns users
+      parameters:
+        - in: query
+          name: age
+          required: true
+          type: number
+      responses:
+        200:
+          description: ok
+        401:
+          $ref: '#/responses/InvalidAuthorization'
+        422:
+          $ref: '#/responses/InvalidPayload'
+responses:
+  InvalidAuthorization:
+    description: InvalidAuthorization
+    schema:
+      type: string
+      example:
+        abc
+  InvalidPayload:
+    description: InvalidPayload
+    schema:
+      type: string
+      example:
+        abc
```

Comment:
might be worth having different example values for things like this just to make sure the body isnt mismatching somehow.

Reply:
I did it for oas3, didn't do it for oas2. Fixed.